src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "My Journey to GSoC @Layer5"
+subtitle: "A Rollercoaster experience"
+date: 2022-06-12 08:00:00 -0630
```

Comment:
Given the anticipated back and forth on grammar reviews. Perhaps, this should be a week from now.

Reply:
@YashKamboj, take extra care in your writing. I will only offer you one additional review after this one. If on your 3rd submission, there are still a number of grammatical errors, we'll forego this post.